src/ctapipe/utils/astro.py

```diff
@@ -4,63 +4,231 @@
 not provided by external packages and/or to satisfy particular needs of
 usage within ctapipe.
 """
-from astropy import units as u
+
+import logging
+from enum import Enum
+from pathlib import Path
+
+import astropy.units as u
 from astropy.coordinates import Angle, SkyCoord
+from astropy.table import Table
+from astropy.time import Time
+
+log = logging.getLogger("main")
 
 __all__ = ["get_bright_stars"]
 
 
-def get_bright_stars(pointing=None, radius=None, magnitude_cut=None):
+CACHE_FILE = Path("~/.psf_stars.ecsv").expanduser()
```

Comment:
This needs to be changed. As discussed in the comments, we want to have the catalogs as resource files in the package and only have the code to create them for developers to update those files if needed.

So this should not be needed at all. *if* we need it, it should be using this function here to obtain an appropriate cache path:
https://github.com/cta-observatory/ctapipe/blob/40faf694867aa570c217c3f9456013684455f6ae/src/ctapipe/utils/download.py#L69

instead of just storing a hidden file in the user's home